src/components/canvas.tsx

```diff
@@ -15,7 +15,7 @@ interface IProps extends IBaseProps {
   readOnly?: boolean;
   document?: DocumentModelType;
   content?: DocumentContentModelType;
-  showEditability?: EditabilityLocation;
+  editabilityLocation?: EditabilityLocation;
```

Comment:
@kswenson you had mentioned this should be called `editabilityLocation` in a previous review to clarify it is not a boolean - looks like I forgot to fix it at the time